BAKKA_Editor/Rendering/SkCircleView.cs

```diff
@@ -26,6 +26,7 @@ internal enum RolloverState
 internal class SkCircleView
```

Comment:
Doesn't really matter, but RolloverState above this and all the mouse variables below this can be deleted since they were either moved or refactored in some way to the cursor